src/sentry/utils/event_frames.py

```diff
@@ -87,7 +140,9 @@ def find_stack_frames(
         # potentially here for backwards compatibility
         frames = get_path(event_data, "stacktrace", "frames", filter=True) or []
         if not frames:
-            threads = get_path(event_data, "threads", "values", filter=True)
+            threads = get_path(event_data, "threads", "values", filter=True) or get_path(
+                event_data, "threads", filter=True
+            )
```

Comment:
This logic was added to support flattened Thread interface frames:
https://develop.sentry.dev/sdk/event-payloads/threads/#examples